web-components/src/components/color-table/ColorTable.ts

```diff
@@ -0,0 +1,85 @@
+import "@/components/table-advanced/TableAdvanced";
```

Comment:
This file is in the components folder - but it is not a reusable component, it is for Storybook only... We should remove it from the components folder

Reply:
Or at least add it to another folder (e.g. Internal/Not Exportable...) or give the folder a prefix... something

Reply:
I did this because it didn't seem to load scss well otherwise. I can try.

Reply:
I have created an `internal-components` folder separate from components. Not it is displayed within a storybook folder called `internal references`.